### PR TITLE
[2.6] Fix forwarded logic

### PR DESCRIPTION
**Pull request by dmcgowan**
Changes from #2219 into release 2.6. Merge into master first.

### CODE REVIEW

**Comment by dmcgowan** [comment]
Added release notes, going to just update the release branch to this commit and create rc.1 tag

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.6.1-rc1 (2017-03-21)
+
+#### Registry
+- Fix Forwarded header handling, revert use of `X-Forwarded-Port`
```
Comment (aaronlehmann) [inline CHANGELOG.md]: Enclose Forwarded in backticks?

Comment (dmcgowan) [inline CHANGELOG.md]: Updated

**Comment by aaronlehmann** [comment]
LGTM

**Pull request merged**